csharp/Platform.Collections/Arrays/CharArrayExtensions.cs

```diff
@@ -24,6 +33,19 @@ public static int GenerateHashCode(this char[] array, int offset, int length)
             return hashAccumulator + (hashSeed * 1566083941);
         }
 
+        /// <summary>
+        /// <para>Checks if all elements of two lists are equal.</para>
+        /// <para>Проверяет равны ли все элементы двух списков.</para>
+        /// </summary>
+        /// <param name="left"><para>The first compared array.</para><para>Первый массив для сравнения.</para></param>
+        /// <param name="leftOffset"><para>Number of displaced elements in first array.</para><para>Количество смещаемых элементов в первом массиве.</para></param>
```

Comment:
Elements are not displaced. Displacement is just an index where the array's segment starts.